modules/quartz-database/src/Volo.Abp.Quartz.Database.EntityFrameworkCore/Volo/Abp/Quartz/Database/EntityFrameworkCore/QuartzDatabaseDbContextModelCreatingExtensions.cs

```diff
@@ -0,0 +1,573 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore;
+using Volo.Abp;
+using Volo.Abp.EntityFrameworkCore.Modeling;
+
+namespace Volo.Abp.Quartz.Database.EntityFrameworkCore
+{
+    public static class QuartzDatabaseDbContextModelCreatingExtensions
+    {
+        public static void ConfigureQuartzDatabase(
+            this ModelBuilder builder,
+            Action<QuartzDatabaseModelBuilderConfigurationOptions> optionsAction = null)
+        {
+            Check.NotNull(builder, nameof(builder));
+
+            //TODO: sqlite database need to create triggers, any way?
```

Comment:
Can you explain in detail?

Reply:
https://github.com/quartznet/quartznet/blob/41846f43abfcfa45c2761a7fb93ac376a6b65d07/database/tables/tables_sqlite.sql#L65-L68

SQLite uses database triggers for cascading deletes.  I did not find a way to automatically use efcore to generate these triggers.

Reply:
Although this PR is closed, I would like to state the results about this comment.

The sqlite connection string uses "Foreign Keys=true" to enable "foreign key". For example: `Data Source=QuartzDatabaseDemo.db;Foreign Keys=true`.

[https://docs.microsoft.com/en-us/dotnet/standard/data/sqlite/connection-strings#foreign-keys](https://docs.microsoft.com/en-us/dotnet/standard/data/sqlite/connection-strings#foreign-keys)